twilio/task_router/__init__.py

```diff
@@ -209,13 +227,15 @@ def __init__(self, account_sid, auth_token, workspace_sid, worker_sid):
                                                          workspace_sid,
                                                          worker_sid)
 
-        self.reservations_url = self.base_url + "/Tasks/**"
         self.activity_url = self.base_url + "/Activities"
+        self.tasks_url = self.base_url + "/Tasks/**"
```

Comment:
I'm not sure this is backwards compatible. If I had an instance of `TaskRouterWorkerCapability`, I would no longer be able to access `reservations_url` with this update, right?